### ISSUE TITLE
Do we need to clarify inactive user interface component

### ISSUE THREAD

**Issue by DavidMacDonald** [w3c/wcag#636]
I did a Twitter poll asking A11y pros to say whether in a tabbed interface the unselected items could be interpreted as inactive and therefore greyed out. There was and 80/20 spit on results.
https://twitter.com/davidmacd/status/1093231012262293504

The question is. Should we allow an inactive user interface be a control that a user clicks on to become active. I would say "no, its not inactive, its unselected".

**Comment by mraccess77** [comment]
I agree -- it's not inactive -- its unselected.   Treating unselected as inactive and ignoring contrast minimums is a very concerning issue and mis-application of the term.

**Comment by awkawk** [comment]
I would say that the unselected tab is part of an ACTIVE UI Component - the Tab Bar - so it needs to meet the ratio.

**Comment by awkawk** [comment]
Following up on this - it seems that 82% of people got this right. As such I don't think that addressing this is a huge priority, but if someone wants to suggest a sentence or two to speak to inactive components the WG can review/accept it.

**Comment by patrickhlauke** [comment]
a small note/clarification in the understanding doc that "inactive user interface component" is to be understood as a disabled/non-actionable component?

**Comment by DavidMacDonald** [comment]
We should probably explain that somewhere.

**Comment by jake-abma** [comment]
Agree when adding / comparing with "disabled":

@patrickhlauke 
> a small note/clarification in the understanding doc that "inactive user interface component" is to be understood as a disabled/non-actionable component

@awkawk 
Being part of a Tar Bar though is not by definition "meet the ratio" as Tab Bars can contain inactive / disabled tabs. (Example: the current tab, two possible clickable tabs (options you already have) and 3 inactive / disabled tabs (options you do not yet have but are available when...)

So the following is not fully correct:

> I would say that the unselected tab is part of an ACTIVE UI Component - the Tab Bar - so it needs to meet the ratio.

**Comment by awkawk** [comment]
@jake-abma good point.

**Comment by awkawk** [comment]
@DavidMacDonald ok, you brought it up - want to suggest some text? :)

**Comment by patrickhlauke** [comment]
made a first verbose stab at it https://github.com/w3c/wcag/pull/638 ... feel free to rip it apart/take it as a starting point 👍

**Comment by alastc** [comment]
It's a pretty big addition for the glossary, and the 'inactive' part isn't included in the definition term. would it be better in the understanding document(s)? It is only used for the contrast ones.

**Comment by patrickhlauke** [comment]
it's indeed just the contrast ones, but all of them - 1.4.3, 1.4.6, 1.4.11

could be added as a note to the understanding for all three of them. but conceptually i'd tend to want to see that defined in the glossary where it explains what a user interface component actually is (as it defines a particular state/characteristic of a user interface component). not overly married to the idea though either way, just think it'd avoid confusion (having to go from the normative SC text, then to the glossary to see what a "user interface component" in general is, then back to the SC and its related understanding to work out what's actually meant by an "inactive" component)

**Comment by alastc** [comment]
Yea, it's just kinda big for the location. We also already have a section on it in the 1.4.11 understanding doc.

Just seen [Michael's suggestion](https://github.com/w3c/wcag/pull/638#issuecomment-467654234), maybe that's a better option.

For a glossary item, I'd try to keep it short, with something like:

> **Inactive user interface components:** 
> User interface components that visible but not operable.
> Example: a submit button at the bottom of a form that is visible but cannot be activated until all the required fields in the form are completed.
> Note: Inactive is not the same as not selected. A tab component with a selected tab and others which are not selected are all active.

**Comment by patrickhlauke** [comment]
sure, i could live with that as well (assuming there's a cross-reference between the two definitions)

**Comment by patrickhlauke** [comment]
as the question about "what does 'inactive' actually mean?" has come up recently, and I was reminded of our work nearly 4 years ago ... any chance this (and maybe even related ones like https://github.com/w3c/wcag/pull/1547) could be looked at at some point @alastc @awkawk

**Comment by giacomo-petri** [comment]
Also related to https://github.com/w3c/wcag/issues/4874#issuecomment-3150513328 (contrast exception for inactive user interface component)
